### PR TITLE
docs: add migration guide for cards from V0 to V9

### PR DESCRIPTION
**Pull request by marcosmoura**
Move migration guide from plain .md file to a storybook example.

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<details>
<summary>Unchanged fixtures</summary>

| Package & Exports                                                                                                                                                             |          Size (minified/GZIP) |
| ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------------------------: |
| <samp>react-card</samp> <br /> <abbr title='bundle-size/Card.All.fixture.js'>Card - All</abbr>                                                                                |  `77.975 kB`<br />`22.496 kB` |
| <samp>react-card</samp> <br /> <abbr title='bundle-size/Card.fixture.js'>Card</abbr>                                                                                          |   `72.92 kB`<br />`21.108 kB` |
| <samp>react-card</samp> <br /> <abbr title='bundle-size/CardFooter.fixture.js'>CardFooter</abbr>                                                                              |     `9.04 kB`<br />`3.807 kB` |
| <samp>react-card</samp> <br /> <abbr title='bundle-size/CardHeader.fixture.js'>CardHeader</abbr>                                                                              |   `10.805 kB`<br />`4.443 kB` |
| <samp>react-card</samp> <br /> <abbr title='bundle-size/CardPreview.fixture.js'>CardPreview</abbr>                                                                            |    `9.847 kB`<br />`4.159 kB` |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/ButtonProviderAndTheme.fixture.js'>react-components: Button, FluentProvider & webLightTheme</abbr>              |  `59.888 kB`<br />`16.652 kB` |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/MultipleComponents.fixture.js'>react-components: Accordion, Button, FluentProvider, Image, Menu, Popover</abbr> | `188.928 kB`<br />`53.088 kB` |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/ProviderAndTheme.fixture.js'>react-components: FluentProvider & webLightTheme</abbr>                            |  `34.379 kB`<br />`11.322 kB` |
| <samp>react-portal-compat</samp> <br /> <abbr title='bundle-size/PortalCompat.fixture.js'>PortalCompatProvider</abbr>                                                         |    `6.069 kB`<br />`2.053 kB` |

</details>
<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/499f0b7118b369e0652f61ca405145ae0bf7f4d8'>499f0b7118b369e0652f61ca405145ae0bf7f4d8</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react-components`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>Avatar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26186/perf-test-react-components/Avatar-mount_base.html">1540</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26186/perf-test-react-components/Avatar-mount.html">1536</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Button</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26186/perf-test-react-components/Button-mount_base.html">1169</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26186/perf-test-react-components/Button-mount.html">1175</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26186/perf-test-react-components/FluentProvider-mount_base.html">2040</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26186/perf-test-react-components/FluentProvider-mount.html">1947</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26186/perf-test-react-components/FluentProviderWithTheme-mount_base.html">777</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26186/perf-test-react-components/FluentProviderWithTheme-mount.html">780</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26186/perf-test-react-components/FluentProviderWithTheme-virtual-rerender_base.html">691</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26186/perf-test-react-components/FluentProviderWithTheme-virtual-rerender.html">702</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26186/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount_base.html">746</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26186/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount.html">788</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>MakeStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26186/perf-test-react-components/MakeStyles-mount_base.html">2296</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26186/perf-test-react-components/MakeStyles-mount.html">2340</a></td>
            <td>50000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26186/perf-test-react-components/Persona-mount_base.html">3685</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26186/perf-test-react-components/Persona-mount.html">3711</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26186/perf-test-react-components/SpinButton-mount_base.html">3158</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26186/perf-test-react-components/SpinButton-mount.html">3082</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9425"/>

**Comment by fabricteam** [comment]
## 🕵 fluentuiv9 No visual regressions between this PR and main
<div id="vrtCommentfluentuiv9"/>

File: apps/public-docsite-v9/src/Concepts/Migration/FromV0/Components/Card.Card.stories.mdx [hunk 1]
```diff
@@ -0,0 +1,227 @@
...
+
+```tsx
+import { Card } from '@fluentui/react-northstar';
+const Component = () => <Card>Lorem ipsum, dolor sit amet consectetur adipisicing elit.</Card>;
+```
+
+After:
+
+```tsx
+import { Card } from '@fluentui/react-components';
```
Comment (Hotell) [inline apps/public-docsite-v9/src/Concepts/Migration/FromV0/Components/Card.Card.stories.mdx:19]:
```suggestion
import { Card } from '@fluentui/react-components/unstable';
```

**Review by Hotell — APPROVED** [review]

**Pull request merged**